cpa-repo/src/main/kotlin/no/nav/emottak/cpa/App.kt

```diff
@@ -47,7 +47,7 @@ fun cpaApplicationModule(cpaDbConfig: HikariConfig, cpaMigrationConfig: HikariCo
             json()
         }
         install(Authentication) {
-            tokenValidationSupport(AZURE_AD_AUTH, Security().config)
+            tokenValidationSupport(AZURE_AD_AUTH, AuthConfig.getCpaRepoConfig())
```

Comment:
Først klassen heter AuthCONFIG etterpå du har "get" og på slutten 1 til Config i navnet:

"Authconfig.cpaRepo" short and clear.